src/brasil/gov/portal/upgrades/v10901/profile/registry.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0"?>
+<registry>
+  <record name="collective.cover.controlpanel.ICoverSettings.layouts">
+    <value>
+      <element key="Layout vazio">[{"type": "row", "children": [{"type": "group", "column-size": 12, "roles": ["Manager"]}]}]</element>
+      <!-- XXX: layout "Destaques" will remain unchanged after upgrade step -->
```

Comment:
@agnogueira @idgserpro I need some feedback on this

Reply:
eu sei, o que quero saber é se deixamos o layout sem modificação para sites migrados.

Reply:
@hvelarde o layout modelo do destaque não é uma capa? Acho que pode sofrer as mesmas transformações das demais capas.